### PR TITLE
Create py_test.yml

### LINKED ISSUE #3: Add pytest coverage
If unsure ask @miroslavpojer for advice.

### PR DESCRIPTION
**Pull request by Zejnilovic**
Closes #3

### CODE REVIEW

File: .github/workflows/column2Vec_test.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: "Column2Vec tests"
+
+on:
+  schedule:
+    - cron: '0 10 * * 6'
```
Comment (Zejnilovic) [inline .github/workflows/column2Vec_test.yml:5]: Why the schedule? Also why not on PR?

File: .github/workflows/column2Vec_test.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  workflow_dispatch:
+
+
+jobs:
+  python-tests:
+    name: Run Python Tests
```
Comment (Zejnilovic) [inline .github/workflows/column2Vec_test.yml:11]: Both workflows should not be named Python Tests. Rename this one

File: .github/workflows/py_test.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        run: |
+          pip install pytest
+          pip install -r requirements.txt
+          pip install pylint
+
+      - name: Run tests
+        run: |
+          pytest test/test_types.py
+          pytest test/test_metadata.py
+          pytest test/test_comparator.py
```
Comment (Zejnilovic) [inline .github/workflows/py_test.yml:32]: Why not just `pytest test/`

File: .github/workflows/column2Vec_test.yml (hunk 1 above)
Comment (OlivieFranklova) [reply .github/workflows/column2Vec_test.yml:5]: It takes one hour to run.

Comment (Zejnilovic) [reply .github/workflows/column2Vec_test.yml:5]: I have a feeling that goes over the limits of free GH Actions. Please verify

Comment (Zejnilovic) [reply .github/workflows/column2Vec_test.yml:5]: But in case of a long run maybe we want to do it on `push: maste`

Comment (OlivieFranklova) [reply .github/workflows/column2Vec_test.yml:5]: It should be 2,000 mins per month [here](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions)

I can rewrite it to `push: master`, but that could be more than once per week.

File: .github/workflows/column2Vec_test.yml (hunk 2 above)
Comment (OlivieFranklova) [reply .github/workflows/column2Vec_test.yml]: Fixed

**Review by Zejnilovic — COMMENTED** [review]
I like it. Cannot approve as I am the PR creator on GitHub. Please merge when you are ready.

**Pull request merged**